test/Serialization/vtable-lookup.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -parse-as-library -module-name Classes -emit-module -emit-module-path %t/Classes.swiftmodule %S/Inputs/classes.swift
+// RUN: %target-build-swift -parse-as-library -module-name Classes -c -o %t/classes.o %S/Inputs/classes.swift
+// RUN: %target-build-swift -I %t %s %t/classes.o -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out | %FileCheck %s
+// REQUIRES: executable_test
```

Comment:
It should be possible to do this with an IRGen test, right? That'd be faster to run.

Reply:
I startet adding more executable tests recently because they cover more aspects of a problem. SIL/IR check tests and executable tests both have advantages and disadvantages. I feel that an executable tests is better for this case - and I think the execution time is not an issue here.